models/vgg.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, vgg_name, batch_norm=False, num_classes=1000, init_weights=Tr
         self.name = vgg_name
         self.bn = batch_norm
         self.classifier = nn.Sequential(
-            nn.Linear(512 * 7 * 7, 4096),
```

Comment:
removed because we use a different dataset, but this will break on the original dataset